stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetPaymentMethodsListFragment.kt

```diff
@@ -1,18 +1,24 @@
 package com.stripe.android.paymentsheet
 
+import android.app.Application
 import android.os.Bundle
 import android.view.View
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.AndroidViewModel
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.stripe.android.R
 import com.stripe.android.databinding.FragmentPaymentsheetPaymentMethodsListBinding
+import com.stripe.android.paymentsheet.model.Selection
 
 internal class PaymentSheetPaymentMethodsListFragment : Fragment(R.layout.fragment_paymentsheet_payment_methods_list) {
```

Comment:
Can we add a test using [FragmentScenario](https://developer.android.com/reference/androidx/fragment/app/testing/FragmentScenario)?

Doesn't have to be in this PR.

Reply:
Sure, I'll do that in a follow up :)